data/mods/DinoMod/monstergroups/lab.json

```diff
@@ -4,20 +4,24 @@
     "name": "GROUP_CENTRAL_LAB",
     "default": "mon_zombie_scientist",
     "monsters": [
```

Comment:
What do these frequencies add up to, BTW? And does this vary by time period?

Reply:
Ha, you got me. I haven't added them. Yes, a bunch of the harder spawns won't appear on day one, and then zombies are phased in and living dinos are phased out.

Reply:
Chuckle... I had actually more meant "does what it adds up to vary by time period".

Reply:
For sure yes. Smallest overall on day one, then I think the most like day three, then smaller again as the living dinos stop appearing